docs/input/docs/getting-started/principles.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Please add some line breaks, the rendered version is incredibly hard to read.
![grafik](https://user-images.githubusercontent.com/25186516/69242919-2e713500-0ba2-11ea-8c33-172e38ff7d23.png)